features/configuration_loading.feature

```diff
@@ -30,19 +30,71 @@ Feature: Offer different ways how to load configuration
 
   Scenario: Configuration file in working directory
     Given the smelly file 'smelly.rb'
-    And a configuration file 'full_mask.reek'
+    And a configuration file '.reek.yml'
     When I run reek smelly.rb
     Then it reports no errors
     And it succeeds
 
   Scenario: Two opposing configuration files and we stop after the first one
     Given the smelly file 'smelly.rb' in a subdirectory
     And a configuration file 'partial_mask.reek' in a subdirectory
-    And a configuration file 'full_mask.reek'
+    And a configuration file '.reek.yml'
+    When I run "reek -c partial_mask.reek smelly.rb" in a subdirectory
+    Then the exit status indicates smells
+    And it reports:
+      """
+      smelly.rb -- 1 warning:
+        [5]:UncommunicativeVariableName: Smelly#x has the variable name 'y' [https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md]
+      """
+
+  Scenario: Two opposing configuration files but we ignore the first one due to an improper name
+    Given the smelly file 'smelly.rb' in a subdirectory
+    And a configuration file 'partial_mask.reek' in a subdirectory
+    And a configuration file '.reek.yml'
     When I run "reek smelly.rb" in a subdirectory
+    Then it reports no errors
+    And it succeeds
+
+  Scenario: Do not use the default config file when we explicitly specify one configuration file
+    Given the smelly file 'smelly.rb'
+    And a file named "config.reek" with:
+      """
+      ---
+      UncommunicativeMethodName:
+        enabled: false
+      """
+    And a file named ".reek.yml" with:
+      """
+      ---
+      UncommunicativeVariableName:
+        enabled: false
+      """
+    When I run reek -c config.reek smelly.rb
     Then the exit status indicates smells
     And it reports:
       """
       smelly.rb -- 1 warning:
         [5]:UncommunicativeVariableName: Smelly#x has the variable name 'y' [https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md]
       """
+
+  Scenario: Prefer the default config file over other configuration files
```

Comment:
.. and I think this is the same as `Two opposing configuration files but we ignore the first one due to an improper name`.

Reply:
There is a subtle difference: Both other scenarios talk about subdirectories. But overall I think you're right, the scenarios are too close together to warrant their existence. I removed the "old ones" and kept the new ones since they are more expressive and make it more obvious what happens.